app/main.py

```diff
@@ -1,6 +1,34 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+API_KEY = os.getenv("API_KEY")
+BASE_URL = "https://api.weatherapi.com/v1/current.json"
+CITY = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    if not API_KEY:
+        raise ValueError("API_KEY environment variable not set")
+    print("Getting weather for city Paris")
+    url = f"{BASE_URL}?key={API_KEY}&q={CITY}&aqi=no"
+
+    response = requests.get(url)
```

Comment:
Use params= parameter instead of constructing the url manually
https://requests.readthedocs.io/en/latest/user/quickstart/#passing-parameters-in-urls